packages/terra-application-layout/src/header/_ApplicationHeader.jsx

```diff
@@ -117,13 +119,14 @@ class ApplicationHeader extends React.Component {
 
   renderToggle() {
     const { layoutConfig, intl } = this.props;
+    const theme = this.context;
 
     if (layoutConfig.toggleMenu) {
       return (
-        <div className={cx('navbar-toggle')}>
+        <div className={classNames(cx('navbar-toggle', theme.className))}>
```

Comment:
The theme will be applied by the parent div https://github.com/cerner/terra-framework/pull/1116/files#diff-e660d321a39cefac3535a9755c8a3fcfR245
```suggestion
        <div className={cx('navbar-toggle')}>
```